src/test-run/command.js

```diff
@@ -10,7 +10,8 @@ export default {
     setStepsSharedData:               'set-steps-shared-data',
     getStepsSharedData:               'get-steps-shared-data',
     getAndUncheckFileDownloadingFlag: 'get-and-uncheck-file-downloading-flag',
-    uncheckFileDownloadingFlag:       'uncheck-file-downloading-flag',
     nativeDialogsInfoSet:             'native-dialogs-info-set',
-    takeScreenshot:                   'take-screenshot'
+    takeScreenshot:                   'take-screenshot',
+    // NOTE: we need this message to support new page unload barier's mechanism
```

Comment:
I think this comment is redundant. [this](https://github.com/DevExpress/testcafe-legacy-api/pull/18/files#diff-a21f56bee759db22062e32250e3df48cR161) is enough